<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1774</width>
    <height>1223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="dateTimeEdit_start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>11</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2014</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>2014</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_5">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>1521</width>
      <height>1101</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>sensor statistics</string>
     </attribute>
     <widget class="QTableWidget" name="doorTable">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>421</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_door">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>doors</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="fanTable">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>481</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_fan">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>fan</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="itemTable">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>310</y>
        <width>481</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_item">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="motionTable">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>90</y>
        <width>461</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>60</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>motion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_motion">
      <property name="geometry">
       <rect>
        <x>1360</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tempTable">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>590</y>
        <width>421</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_temp">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>550</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>560</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>temp</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="lightTable">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>590</y>
        <width>481</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_light">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>550</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>560</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>light sensors</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="lightswitchTable">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>590</y>
        <width>461</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>560</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>light switch</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_lightswitch">
      <property name="geometry">
       <rect>
        <x>1340</x>
        <y>550</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>sensor healh check</string>
     </attribute>
     <widget class="QLabel" name="label_hist">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Histrogram for item</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_hist">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_hist">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="graphWidget" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>1421</width>
        <height>921</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="battery">
     <attribute name="title">
      <string>battery tracking</string>
     </attribute>
     <widget class="QTableWidget" name="battTable">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>591</width>
        <height>921</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>battery percetage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh_batt">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QListWidget" name="battList">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>110</y>
        <width>311</width>
        <height>921</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>80</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>monthly change list</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_end">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>11</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2014</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>2014</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>current datetime</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>start datetime</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
